### ISSUE TITLE
setMisc and getMisc functions do not exist in the latest version (1.6.0)

### ISSUE THREAD

**Issue by metwallys** [gaynorr/AlphaSimR#204]
I was working on week 5 of the course and found out that these two functions used to store additional information no longer exist in the latest version without an alternative. I am not able to reproduce the animal breeding program scripts, maybe unless I explicitly install and use the older version of AlphaSimR.

I wanted to know If this is going to be the case from now on to look for alternative methods to store similar information in the future. Thank you in advance!

**UPDATE:**
It took me a while to read and understand the flipping that happened to misc organisation. However I was able to achieve the same results in the course this way.

# AlphaSimR older versions:

## Assign year of birth for the founders
year = 0
founders = setMisc(x = founders,
                   node = "yearOfBirth",
                   value = year)
head(getMisc(x = founders, node = "yearOfBirth"))

# AlphaSimR v1.6.0:

## Assign year of birth for the founders
year = 0

founders@misc$yearOfBirth = rep(year, nInd(founders))
head(founders@misc$yearOfBirth)

**Comment by gregorgorjanc** [comment]
@metwallys indeed, we have removed `setMisc()` and `getMisc()` in 1.6 since the old approach was tricky to use and very slow. You are now using it correctly!

We are aware that we must update MOOC - the edX platform still uses the older version of AlphaSimR so it still works in the platform.

@gaynorr Maybe we should have added a deprecation message for `setMisc()` and `getMisc()`? Happy to do that if you agree!

**Comment by metwallys** [comment]
@gregorgorjanc thank you very much for your prompt response!

**Comment by gregorgorjanc** [comment]
@gaynorr we have updated the edX course to AlphaSimR 1.6.1 so I think you can close this issue.
